gpu.Dockerfile

```diff
@@ -0,0 +1,74 @@
+# Default CUDA version
+ARG CUDA_VERSION=9.0-cudnn7
```

Comment:
@fantes could you review the default cuda version defined here ?

It probably impacts `-DCUDA_ARCH` cmake options in `build/build.sh` script.

I've been able to compile on my setup using:

```
ARG CUDA_VERSION=10.1-cudnn7
```

Reply:
> à partir de 10.0, il y support pour sm_75, qui concerne les dernières cartes nvidia , cad RTX 2080, Titan RTX, Quadro R8000
> forcer 10.0 / 10.1 par rapport à 9.0 n'enlève pas de cartes par contre